lib/states/statesInRedis.js

```diff
@@ -129,13 +129,13 @@ class StateRedis {
             this.client.disconnect();
             ignoreErrors = true;
 
-            this.log.silly('Initiate Fallback to SocketIo States');
+            this.log.silly(this.namespace + ' Initiate Fallback to SocketIo States');
```

Comment:
isn't that spelled like `socket.io`, not `SocketIo`?

Reply:
Done, changed to socket.io